diff -uNr wine-doors.org/src/preferences.py wine-doors/src/preferences.py
--- wine-doors.org/src/preferences.py	2006-12-21 08:52:46.000000000 +0000
+++ wine-doors/src/preferences.py	2006-12-24 05:47:27.000000000 +0000
@@ -90,7 +90,7 @@
 		
 	def Default( self ):
 		if os.environ['USER'] == "root":
-			self['wineroot'] = "/usr/share/wine/wineroot"
+			self['wineroot'] = "/usr/share/wine"
 		else:
 			self['wineroot'] = os.environ['HOME'] + "/.wine"
 		self['winebinary'] = os.popen( "which wine" ).read().rstrip()
